lib/scrum_lint/configuration.rb

```diff
@@ -63,7 +63,7 @@ def config_file_contents
     end
 
     def config_file_path
-      './.scrum-lint.yml'
+      ENV.fetch('config_file_path', './.scrum-lint.yml')
```

Comment:
Allows the user to specify where the `.scrum-lint.yml` file is located. This helps the test specs not use the real config file.